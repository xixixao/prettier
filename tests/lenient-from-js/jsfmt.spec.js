run_spec(__dirname, ["flow"], {
  bracketSpacing: false,
  lenient: true,
  semi: false
});
