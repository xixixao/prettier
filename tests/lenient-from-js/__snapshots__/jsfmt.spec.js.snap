// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`arrows.js 1`] = `
[1, 2, 3].forEach(x => {
  fulemasdosuahdsaoudhsaoudhsadoadsdsadsdsaoduhsaouhdaouh(x);
});

[1, 2, 3].forEach(x =>
  fulemasdosuahdsaoudhsaoudhsadoadsdsadsdsaoduhsaouhdaouh(x),
);

f(x =>{
  while (true) {
    g();
  }
})
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[1, 2, 3].forEach(x =>
  fulemasdosuahdsaoudhsaoudhsadoadsdsadsdsaoduhsaouhdaouh(x);
)

[1, 2, 3].forEach(x =>
  fulemasdosuahdsaoudhsaoudhsadoadsdsadsdsaoduhsaouhdaouh(x)
)

f(x =>
  while true
    g()
)

`;

exports[`classes.js 1`] = `
class X extends B {
  other() {
    return da;
  }

  method() {
    return bla;
  }

  m = () => {
    b();
  }
}

f(class X{
  m(){
    bla()
  }
})
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
class X extends B
  other()
    return da

  method()
    return bla

  m = () =>
    b();

f(
  class X
    m()
      bla();
)

`;

exports[`loops.js 1`] = `
while (x) {
  a()
}

for (let i = 0; i < cubes.length; i++) {
  alert(i, cubes[i]);
}

switch (x) {
  case 'a': a(); break;
}
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
while x
  a()

for let i = 0; i < cubes.length; i++
  alert(i, cubes[i])

switch x
  case "a"
    a()
    break


`;

exports[`trycatch.js 1`] = `
try {} catch (x) {} finally {}
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
try {} catch (x) {} finally {}

`;
